### PR TITLE
Proposal 001 - Guru Network Core Contributors Incentives and Governance 

### PR DESCRIPTION
**Pull request by evahteev**
# [PROPOSAL-001] Guru Network Core Contributors(CC) Incentives and Governance 

## Summary

This proposal outlines the incentive distribution plan for the future Guru Network Core Contributors(CC) DAO, focusing on community engagement, core contributor rewards. The aim is to ensure transparency, accountability, and sustained growth for the Guru Network.

## Background

The Guru Network has consistently delivered high-impact products, including the Block Explorer, Data Warehouse, Flow, DexGuru V2, and Terminal integration, within a year. Our team, composed of seasoned Web3 professionals, has navigated both the highs and lows of the DeFi landscape. As we prepare for the next phase of growth, establishing a structured governance and incentive framework is crucial to maintaining our momentum and fostering a collaborative ecosystem.

### Incentive Distribution Plan

To ensure the long-term prosperity of the Guru Network, we propose the following:

1. **Create CC Seasons** and align them with Community Incentives Seasons in terms of periods and distributed allocation.
2. **Setup a CC Governance Framework** tasked with developing and proposing a Community Governance Framework before the Guru Network Mainnet launch.

Out of the total token supply, 40% is allocated for distribution between Community and CC as follows:

- 
- **20% for Guru Seasons 1-5 Community:**  
  Rewarding Season Pass NFT holders and active community participants. The distribution is divided into seasons, with Seasons 1-2 are gaining and boosting rewards by participation now (Testnet phase). Activities snapshot will be done close to the Mainnet launch. Community Seasons 3-5 planned for the next 24 months as a continuous community incentives aligned with CC Seasons.

- **20% for CC:**  
  Divided into seasonal allocations starting now with the Guru Network testnet phase. Governance and voting begin with CC, evolving into a polished Community Governance framework by the Mainnet launch. The current list of CC can be found here: [CC](../CORE_CONTRIBUTORS.md).

### CC Incentive Distribution Mechanism

**Proposed unlocking system for the CC’ allocation:**

- **First Unlock:**  
  In six months starting from TGE, 5% of the total supply (of 20% CC total) will be unlocked. Distribution will follow a traditional half-year/yearly review system based on contributions during the Testnet phase, Mainnet Launch, and Ecosystem Growth.  

- **Subsequent Unlocks:**  
  Every six months, another 5% will unlock, maintaining rewards based on continuous performance. CC will vote for claim and distribution in 6-month periods further down the line, making it a total of 4 CC Seasons.

### Community Incentive Distribution Mechanism

The community’s 20% allocation will be unlocked in 4 phases (Seasons 1, 2; 3; 4; 5), encouraging ongoing engagement and contributions. A total of four community distributions are aligned with the same CC seasons. Eligible community members include Season Pass NFT holders and active participants.

### Governance Framework

The Community and CC Governance framework will be adjusted and proposed for discussion before Mainnet Launch. The framework proposed for Season 1 ([GuruDAO Bootstrap Guide](https://docs.dex.guru/gurudao/gurudao-bootstrap-guide)) will be adjusted to modern-day realities. The process will be public to ensure that no interested parties can back off those decisions without public consequences.

A Multisig Safe with a balanced voting system will be used to provide every core contributor with a voice and voting rights. This system, combined with our Core Contribotors Seasons planned for the next two years, creates long-term incentives for the CC team.

### List of CC

[CC](../CORE_CONTRIBUTORS.md)

## Specifications

N/A

## Pros

- Encourages active CC engagement and contributions, providing long-term incentives aligned with community ones.
- Encourages active community engagement and contributions.
- Provides a structured and transparent reward system for CC.

## Cons

- Requires careful management to ensure fair distribution and governance.
- Potential challenges in maintaining consistent engagement and contributions from the community.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,6 @@
-# governance-framework
-Guru Network Governance Framework
+# Guru Network Governance Framework
+
+The Guru Network has consistently delivered high-impact products, including the Block Explorer, Data Warehouse, Flow, DexGuru V2, and Terminal integration, within a year. Our team, composed of seasoned Web3 professionals, has navigated both the highs and lows of the DeFi landscape. As we prepare for the next phase of growth, establishing a structured governance and incentive framework is crucial to maintaining our momentum and fostering a collaborative ecosystem.
+
+
+[Guru Network Core Contributiors List](CORE_CONTRIBUTORS.md)
```
Comment (decentralizee) [inline README.md:6]: typo - in Contribut**i**ors

**Review by sudlenkovea — APPROVED** [review]

File: proposals/001_Core_Contributors_governance.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+To ensure the long-term prosperity of the Guru Network, we propose the following:
+
+1. **Create Core Contributors Seasons** and align them with Community Incentives Seasons in terms of periods and distributed allocation.
+2. **Setup a Core Contributors Governance Framework** tasked with developing and proposing a Community Governance Framework before the Guru Network Mainnet launch.
+
+Out of the total token supply, 60% is allocated for distribution as follows:
+
+- **20% for Guru Seasons 1-5 Community:**  
+  Rewarding Season Pass NFT holders and active community participants. The distribution is divided into seasons, with Seasons 1-2 are gaining and boosting rewards by participation now (Testnet phase). Activities snapshot is done close to the Mainnet launch. Community Seasons 3-5 planned for the next 24 months as a continuous community incentives aligned with Core Contributors Seasons.
```
Comment (decentralizee) [inline proposals/001_Core_Contributors_governance.md:23]: Activities snapshot is done close to the Mainnet launch

will be done

**Review by oushanskeen — APPROVED** [review]

**Review by shestakoven — APPROVED** [review]
LGTM

**Review by kelar86 — APPROVED** [review]

File: proposals/001_Core_Contributors_governance.md (hunk 2 above)
Comment (evahteev) [reply proposals/001_Core_Contributors_governance.md]: done

File: proposals/001_Core_Contributors_governance.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
+# [PROPOSAL-001] Guru Network Core Contributors Incentives and Governance 
+
+## Summary
+
+This proposal outlines the incentive distribution plan for the Guru Network Core Contributors DAO, focusing on community engagement, core contributor rewards. The aim is to ensure transparency, accountability, and sustained growth for the Guru Network.
```
Comment (decentralizee) [inline proposals/001_Core_Contributors_governance.md:5]: Should we name Core Contributors with capitals across ALL document?

Should we introduce who are Core Contributors, and show CC's list in the beginning of the document?

**Review by GenkaOm — APPROVED** [review]

**Review by Kharabet — APPROVED** [review]

**Review by NemeZZiZZ — APPROVED** [review]

**Comment by nashirudin** [comment]
agrred

**Pull request merged**